DOCS-8676 Tests in Containers Doc Update (#24559)
DataDog/documentation: content/en/tests/containers.md

| `BITRISE_BUILD_NUMBER`              | Build number of the build on bitrise.io.                                                                         |
| `BITRISE_BUILD_URL`                 | The URL of the build on bitrise.io.                                                                              |
| `BITRISE_SOURCE_DIR`                | Path to the base working directory. By default, it’s the directory where Bitrise runs, unless you provide a different value. |
| `GIT_REPOSITORY_URL`                | The URL of the Git repository that hosts your app.                                                               |
| `BITRISE_GIT_COMMIT`                | The commit hash of the Git commit that triggered the build, when applicable.                                     |
| `GIT_CLONE_COMMIT_HASH`             | The hash of the commit that the build uses (the cloned commit).                                                  |
| `BITRISEIO_GIT_BRANCH_DEST`         | The destination or target branch of the pull request that triggered the build, used only with builds triggered by pull requests. |

> | `BITRISE_GIT_BRANCH`                | The git branch that is built by Bitrise, for example, `main`.                                                    |

Reviewer: ```suggestion
| `BITRISE_GIT_BRANCH`                | The Git branch that is built by Bitrise, for example, `main`.                                                    |
```

|-------------------------------|-------------------------------------------------------------------------------------------------|
| `BUILDKITE`                   | Always true.                                                                                   |
| `BUILDKITE_PIPELINE_SLUG`     | The pipeline slug on Buildkite as used in URLs.                                                 |
| `BUILDKITE_JOB_ID`            | The internal UUID Buildkite uses for this job.                                                   |
| `BUILDKITE_BUILD_ID`          | The UUID of the build.                                                                          |
| `BUILDKITE_BUILD_NUMBER`      | The build number. This number increases with every build and is unique within each pipeline.   |
| `BUILDKITE_BUILD_URL`         | The URL for this build on Buildkite.                                                             |
| `BUILDKITE_BUILD_CHECKOUT_PATH` | The path where the agent has checked out your code for this build.                             |
| `BUILDKITE_REPO`              | The repository of your pipeline.                                                                  |

> | `BUILDKITE_COMMIT`            | The git commit object of the build.                                                              |

Reviewer: ```suggestion
| `BUILDKITE_COMMIT`            | The Git commit object of the build.                                                              |
```

| `TRAVIS_BUILD_ID`            | The ID of the current build used internally by Travis CI.                                             |
| `TRAVIS_BUILD_NUMBER`        | The number of the current build. For example: `4`.                                                    |
| `TRAVIS_BUILD_WEB_URL`       | URL to the build log.                                                                                 |
| `TRAVIS_BUILD_DIR`           | The absolute path to the directory where the repository being built has been copied on the worker.    |
| `TRAVIS_JOB_WEB_URL`         | URL to the job log.                                                                                   |
| `TRAVIS_REPO_SLUG`           | The slug (in form: `owner_name/repo_name`) of the repository currently being built.                   |
| `TRAVIS_COMMIT`              | The commit that the current build is testing.                                                         |
| `TRAVIS_BRANCH`              | For push builds, the name of the branch. For PR builds, the name of the branch targeted by the PR.    |

> | `TRAVIS_TAG`                 | If the current build is for a git tag, this variable is set to the tag’s name, otherwise it is empty. |

Reviewer: ```suggestion
| `TRAVIS_TAG`                 | If the current build is for a Git tag, this variable is set to the tag’s name, otherwise it is empty. |
```

| `TRAVIS_COMMIT_MESSAGE`      | The commit subject and body, unwrapped.                                                               |

For a comprehensive list of environment variables set by Travis CI for every build, see the [official Travis CI documentation][101].

[101]: https://docs.travis-ci.com/user/environment-variables/#default-environment-variables

{{% /tab %}}
{{% tab "Buddy CI" %}}

| Environment Variable                                | Description                                                                                           |
|-----------------------------------------|-------------------------------------------------------------------------------------------------------|
| `BUDDY`                                 | Represents whether the current environment is a Buddy environment. For example: `true`.    |
| `BUDDY_SCM_URL`                         | The URL of the repository synchronized with the project. For example: `https://github.com/githubaccount/repository`. |
| `BUDDY_EXECUTION_REVISION`              | The SHA1 hash of the commit of the current pipeline run. For example: `46c360492d6372e5335300776806af412755871`. |

> | `BUDDY_EXECUTION_BRANCH`                | The name of the Git branch of the current pipeline run. For example: `master`.             |

Reviewer: ```suggestion
| `BUDDY_EXECUTION_BRANCH`                | The name of the Git branch of the current pipeline run. For example: `main`.             |
```
I left the Jenkins instances, but this one seems fine to change?

| `CF_PIPELINE_NAME`           | The full path of the pipeline, including the project to which it is assigned, if any.                 |
| `CF_BUILD_URL`               | The URL to the build in Codefresh.                                                                    |
| `CF_STEP_NAME`               | The name of the step, for example, "MyUnitTests".                                                     |
| `CF_BRANCH`                  | The branch name or tag of the Git repository associated with the main pipeline at the time of execution. |
| `CF_REVISION`                | The revision of the Git repository of the main pipeline, at the time of execution.                   |

For a comprehensive list of environment variables set by Codefresh for every build, see the [official Codefresh documentation][101].

[101]: https://codefresh.io/docs/docs/pipelines/variables/

{{% /tab %}}
{{% tab "GitHub Actions" %}}

| Environment Variable       | Description                                                                                           |
|----------------------------|-------------------------------------------------------------------------------------------------------|

> | `CI`                       | Always set to `true`, indicating a CI environment.                                                   |

Author: ```suggestion
```

| `GITHUB_ACTION`            | The name of the action currently running, or the id of a step. For example: `repo-owner_name-of-action-repo`. |
| `GITHUB_ACTION_PATH`       | The path where an action is located (only supported in composite actions).                           |
| `GITHUB_ACTION_REPOSITORY` | The owner and repository name of the action being executed. For example: `actions/checkout`.              |
| `GITHUB_ACTIONS`           | Always set to `true` when GitHub Actions is running the workflow.                                    |
| `GITHUB_ACTOR`             | The name of the person or app that initiated the workflow. For example: `octocat`.                       |
| `GITHUB_ACTOR_ID`          | The account ID of the person or app that triggered the workflow. For example: `1234567`.                 |

> | `GITHUB_API_URL`           | The API URL. For example: `https://api.github.com`.                                                      |

Reviewer 2: This list is a lot longer than the list that we originally had. Most of the new environment variables are never used by tracers, for example `GITHUB_API_URL`, or `RUNNER_` vars.

Maybe let's keep in this list only the original environment variables that are actually used by tracers?

| `GITHUB_BASE_REF`          | The base ref or target branch of the pull request (only set for `pull_request` or `pull_request_target` events). For example: `main`. |
| `GITHUB_ENV`               | The path to the file that sets environment variables from workflow commands.                         |
| `GITHUB_EVENT_NAME`        | The name of the event that triggered the workflow. For example: `workflow_dispatch`.                     |
| `GITHUB_EVENT_PATH`        | The path to the file containing the full event webhook payload.                                      |
| `GITHUB_GRAPHQL_URL`       | The GraphQL API URL. For example: `https://api.github.com/graphql`.                                      |
| `GITHUB_HEAD_REF`          | The head ref or source branch of the pull request (only set for `pull_request` or `pull_request_target` events). For example: `feature-branch-1`. |
| `GITHUB_JOB`               | The job ID of the current job. For example: `greeting_job`.                                              |

> | `GITHUB_OUTPUT`            | The path to the file that sets the current step's outputs from workflow commands.                    |
> | `GITHUB_PATH`              | The path to the file that sets system PATH variables from workflow commands.                         |
> | `GITHUB_REF`               | The fully-formed ref of the branch or tag that triggered the workflow. For example: `refs/heads/feature-branch-1`. |
> | `GITHUB_REF_NAME`          | The short ref name of the branch or tag that triggered the workflow. For example: `feature-branch-1`.     |
> | `GITHUB_REF_PROTECTED`     | `true` if branch protections or rulesets are configured for the ref that triggered the workflow run. |
> | `GITHUB_REF_TYPE`          | The type of ref that triggered the workflow. Valid values are `branch` or `tag`.                     |
> | `GITHUB_REPOSITORY`        | The owner and repository name. For example: `octocat/Hello-World`.                                        |
> | `GITHUB_REPOSITORY_ID`     | The ID of the repository. For example: `123456789`.                                                      |
> | `GITHUB_REPOSITORY_OWNER`  | The repository owner's name. For example: `octocat`.                                                     |
> | `GITHUB_REPOSITORY_OWNER_ID` | The repository owner's account ID. For example: `1234567`.                                              |
> | `GITHUB_RETENTION_DAYS`    | The number of days that workflow run logs and artifacts are kept. For example: `90`.                     |
> | `GITHUB_RUN_ATTEMPT`       | A unique number for each attempt of a particular workflow run. For example: `3`.                         |
> | `GITHUB_RUN_ID`            | A unique number for each workflow run within a repository. For example: `1658821493`.                    |
> | `GITHUB_RUN_NUMBER`        | A unique number for each run of a particular workflow in a repository. For example: `3`.                 |
> | `GITHUB_SERVER_URL`        | The URL of the GitHub server. For example: `https://github.com`.                                         |
> | `GITHUB_SHA`               | The commit SHA that triggered the workflow. For example: `ffac537e6cbbf934b08745a378932722df287a53`.    |
> | `GITHUB_STEP_SUMMARY`      | The path to the file that contains job summaries from workflow commands.                             |
> | `GITHUB_TRIGGERING_ACTOR`  | The username of the user that initiated the workflow run.                                            |
> | `GITHUB_WORKFLOW`          | The name of the workflow. For example: `My test workflow`.                                               |
> | `GITHUB_WORKFLOW_REF`      | The ref path to the workflow. For example: `octocat/hello-world/.github/workflows/my-workflow.yml@refs/heads/my_branch`. |
> | `GITHUB_WORKFLOW_SHA`      | The commit SHA for the workflow file.                                                                 |
> | `GITHUB_WORKSPACE`         | The default working directory on the runner for steps. For example: `/home/runner/work/my-repo-name/my-repo-name`. |
> | `RUNNER_ARCH`              | The architecture of the runner executing the job. Possible values are `X86`, `X64`, `ARM`, `ARM64`. |
> | `RUNNER_DEBUG`             | Set to `1` if debug logging is enabled.                                                             |
> | `RUNNER_ENVIRONMENT`       | The environment of the runner executing the job. Possible values are `github-hosted` or `self-hosted`. |
> | `RUNNER_NAME`              | The name of the runner executing the job. For example: `Hosted Agent`.                                   |
> | `RUNNER_OS`                | The operating system of the runner executing the job. Possible values are `Linux`, `Windows`, `macOS`.|
> | `RUNNER_TEMP`              | The path to a temporary directory on the runner. For example: `D:\a\_temp`.                             |
> | `RUNNER_TOOL_CACHE`        | The path to the directory containing preinstalled tools for GitHub-hosted runners. For example: `C:\hostedtoolcache\windows`. |

Author: ```suggestion
```

> | Environment Variable       | Description                                                                                           |
> |----------------------------|-------------------------------------------------------------------------------------------------------|
> | `CI`                       | Always set to `true`, indicating a CI environment.                                                   |
> | `GITHUB_ACTION`            | The name of the action currently running, or the id of a step. For example: `repo-owner_name-of-action-repo`. |
> | `GITHUB_ACTION_PATH`       | The path where an action is located (only supported in composite actions).                           |
> | `GITHUB_ACTION_REPOSITORY` | The owner and repository name of the action being executed. For example: `actions/checkout`.              |
> | `GITHUB_ACTIONS`           | Always set to `true` when GitHub Actions is running the workflow.                                    |
> | `GITHUB_ACTOR`             | The name of the person or app that initiated the workflow. For example: `octocat`.                       |
> | `GITHUB_ACTOR_ID`          | The account ID of the person or app that triggered the workflow. For example: `1234567`.                 |
> | `GITHUB_API_URL`           | The API URL. For example: `https://api.github.com`.                                                      |
> | `GITHUB_BASE_REF`          | The base ref or target branch of the pull request (only set for `pull_request` or `pull_request_target` events). For example: `main`. |
> | `GITHUB_ENV`               | The path to the file that sets environment variables from workflow commands.                         |
> | `GITHUB_EVENT_NAME`        | The name of the event that triggered the workflow. For example: `workflow_dispatch`.                     |
> | `GITHUB_EVENT_PATH`        | The path to the file containing the full event webhook payload.                                      |
> | `GITHUB_GRAPHQL_URL`       | The GraphQL API URL. For example: `https://api.github.com/graphql`.                                      |
> | `GITHUB_HEAD_REF`          | The head ref or source branch of the pull request (only set for `pull_request` or `pull_request_target` events). For example: `feature-branch-1`. |
> | `GITHUB_JOB`               | The job ID of the current job. For example: `greeting_job`.                                              |
> | `GITHUB_OUTPUT`            | The path to the file that sets the current step's outputs from workflow commands.                    |
> | `GITHUB_PATH`              | The path to the file that sets system PATH variables from workflow commands.                         |
> | `GITHUB_REF`               | The fully-formed ref of the branch or tag that triggered the workflow. For example: `refs/heads/feature-branch-1`. |
> | `GITHUB_REF_NAME`          | The short ref name of the branch or tag that triggered the workflow. For example: `feature-branch-1`.     |
> | `GITHUB_REF_PROTECTED`     | `true` if branch protections or rulesets are configured for the ref that triggered the workflow run. |
> | `GITHUB_REF_TYPE`          | The type of ref that triggered the workflow. Valid values are `branch` or `tag`.                     |
> | `GITHUB_REPOSITORY`        | The owner and repository name. For example: `octocat/Hello-World`.                                        |
> | `GITHUB_REPOSITORY_ID`     | The ID of the repository. For example: `123456789`.                                                      |
> | `GITHUB_REPOSITORY_OWNER`  | The repository owner's name. For example: `octocat`.                                                     |
> | `GITHUB_REPOSITORY_OWNER_ID` | The repository owner's account ID. For example: `1234567`.                                              |
> | `GITHUB_RETENTION_DAYS`    | The number of days that workflow run logs and artifacts are kept. For example: `90`.                     |
> | `GITHUB_RUN_ATTEMPT`       | A unique number for each attempt of a particular workflow run. For example: `3`.                         |
> | `GITHUB_RUN_ID`            | A unique number for each workflow run within a repository. For example: `1658821493`.                    |
> | `GITHUB_RUN_NUMBER`        | A unique number for each run of a particular workflow in a repository. For example: `3`.                 |
> | `GITHUB_SERVER_URL`        | The URL of the GitHub server. For example: `https://github.com`.                                         |
> | `GITHUB_SHA`               | The commit SHA that triggered the workflow. For example: `ffac537e6cbbf934b08745a378932722df287a53`.    |
> | `GITHUB_STEP_SUMMARY`      | The path to the file that contains job summaries from workflow commands.                             |
> | `GITHUB_TRIGGERING_ACTOR`  | The username of the user that initiated the workflow run.                                            |
> | `GITHUB_WORKFLOW`          | The name of the workflow. For example: `My test workflow`.                                               |
> | `GITHUB_WORKFLOW_REF`      | The ref path to the workflow. For example: `octocat/hello-world/.github/workflows/my-workflow.yml@refs/heads/my_branch`. |
> | `GITHUB_WORKFLOW_SHA`      | The commit SHA for the workflow file.                                                                 |
> | `GITHUB_WORKSPACE`         | The default working directory on the runner for steps. For example: `/home/runner/work/my-repo-name/my-repo-name`. |
> | `RUNNER_ARCH`              | The architecture of the runner executing the job. Possible values are `X86`, `X64`, `ARM`, `ARM64`. |
> | `RUNNER_DEBUG`             | Set to `1` if debug logging is enabled.                                                             |
> | `RUNNER_ENVIRONMENT`       | The environment of the runner executing the job. Possible values are `github-hosted` or `self-hosted`. |
> | `RUNNER_NAME`              | The name of the runner executing the job. For example: `Hosted Agent`.                                   |
> | `RUNNER_OS`                | The operating system of the runner executing the job. Possible values are `Linux`, `Windows`, `macOS`.|
> | `RUNNER_TEMP`              | The path to a temporary directory on the runner. For example: `D:\a\_temp`.                             |
> | `RUNNER_TOOL_CACHE`        | The path to the directory containing preinstalled tools for GitHub-hosted runners. For example: `C:\hostedtoolcache\windows`. |

Author: ```suggestion
| Environment Variable       | Description                                                                                           |
|----------------------------|-------------------------------------------------------------------------------------------------------|
| `GITHUB_ACTION`            | The name of the action currently running, or the ID of a step. For example: `repo-owner_name-of-action-repo`. |
| `GITHUB_SERVER_URL`        | The URL of the GitHub server. For example: `https://github.com`.                                       |
| `GITHUB_RUN_ID`            | A unique number for each workflow run within a repository. For example: `1658821493`.                 |
| `GITHUB_RUN_NUMBER`        | A unique number for each run of a particular workflow in a repository. For example: `3`.              |
| `GITHUB_RUN_ATTEMPT`       | A unique number for each attempt of a particular workflow run. For example: `3`.                      |
| `GITHUB_WORKFLOW`          | The name of the workflow. For example: `My test workflow`.                                            |
| `GITHUB_WORKSPACE`         | The default working directory on the runner for steps. For example: `/home/runner/work/my-repo-name/my-repo-name`. |
| `GITHUB_REPOSITORY`        | The owner and repository name. For example: `octocat/Hello-World`.                                    |
| `GITHUB_SHA`               | The commit SHA that triggered the workflow. For example: `ffac537e6cbbf934b08745a378932722df287a53`. |
| `GITHUB_HEAD_REF`          | The head ref or source branch of the pull request (only set for `pull_request` or `pull_request_target` events). For example: `feature-branch-1`. |
| `GITHUB_REF`               | The fully-formed ref of the branch or tag that triggered the workflow. For example: `refs/heads/feature-branch-1`. |
| `GITHUB_JOB`               | The job ID of the current job. For example: `greeting_job`.                                           |
```

For a comprehensive list of environment variables set by GitHub Actions for every build, see the [official GitHub documentation][101].

[101]: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/variables#default-environment-variables

{{% /tab %}}
{{% tab "GitLab CI" %}}

> | Variable              | Description                                                                                              |

Author: ```suggestion
| Environment Variable              | Description                                                                                              |
```

| `CI_COMMIT_SHA`       | The commit revision the project is built for.                                                            |
| `CI_COMMIT_REF_NAME`  | The branch or tag name for which project is built.                                                        |
| `CI_COMMIT_BRANCH`    | The commit branch name. Available in branch pipelines.                                                   |
| `CI_COMMIT_TAG`       | The commit tag name. Available only in pipelines for tags.                                               |
| `CI_COMMIT_AUTHOR`    | The author of the commit in Name <email> format.                                                         |
| `CI_COMMIT_MESSAGE`   | The full commit message.                                                                               |
| `CI_COMMIT_TIMESTAMP` | The timestamp of the commit in the ISO 8601 format. For example, 2022-01-31T16:47:55Z. UTC by default.  |

For a comprehensive list of environment variables set by GitLab CI for every build, see the [official GitLab documentation][101].

[101]: https://docs.gitlab.com/ee/ci/variables/predefined_variables.html
{{% /tab %}}
{{% tab "Jenkins" %}}

> | Variable              | Description                                                                                              |

Author: ```suggestion
| Environment Variable              | Description                                                                                              |
```

| `GIT_COMMIT`          | The Git hash of the commit checked out for the build.                                                    |
| `GIT_BRANCH`          | The Git branch that was checked out for the build.                                                       |
| `NODE_NAME`           | The name of the node the build is running on. Equals 'master' for the master node.                       |
| `NODE_LABELS`         | A comma-separated list of labels assigned to the node.                                                   |
| `DD_CUSTOM_TRACE_ID`  | Custom variable set by the Jenkins Datadog Plugin for trace IDs.                                        |
| `DD_CUSTOM_PARENT_ID` | Custom variable set by the Jenkins Datadog Plugin for parent IDs.                                        |

For a comprehensive list of environment variables set by Jenkins for every build, see the [official Jenkins documentation][101].

[101]: https://www.jenkins.io/doc/book/pipeline/jenkinsfile/#using-environment-variables
[102]: https://github.com/jenkinsci/datadog-plugin

{{% /tab %}}
{{% tab "TeamCity" %}}

> | Variable                     | Description                                                                                                  |

Author: ```suggestion
| Environment Variable                     | Description                                                                                                  |
```

| `TRAVIS_JOB_WEB_URL`         | URL to the job log.                                                                                  |
| `TRAVIS_REPO_SLUG`           | The slug (in form: owner_name/repo_name) of the repository currently being built.                    |
| `TRAVIS_COMMIT`              | The commit that the current build is testing.                                                         |
| `TRAVIS_BRANCH`             | For push builds, the name of the branch. For PR builds, the name of the branch targeted by the PR.   |
| `TRAVIS_TAG`                 | If the current build is for a git tag, this variable is set to the tag’s name, otherwise it is empty.|
| `TRAVIS_PULL_REQUEST_SLUG`   | If the current job is a pull request, the slug of the repository from which the PR originated.        |
| `TRAVIS_PULL_REQUEST_BRANCH` | If the current job is a pull request, the name of the branch from which the PR originated.            |
| `TRAVIS_COMMIT_MESSAGE`      | The commit subject and body, unwrapped.                                                               |

> | `TRAVIS_ALLOW_FAILURE`       | Set to `true` if the job is allowed to fail, `false` otherwise.                                       |

Reviewer 2: Same as for Github Actions, this is not used by tracers

> | Variable                     | Description                                                                                           |
> |------------------------------|-------------------------------------------------------------------------------------------------------|
> | `TRAVIS`                     | Always set to `true` to indicate that the build is running on Travis CI.                            |
> | `TRAVIS_BUILD_ID`            | The ID of the current build used internally by Travis CI.                                            |
> | `TRAVIS_BUILD_NUMBER`        | The number of the current build (such as “4”).                                                         |
> | `TRAVIS_BUILD_WEB_URL`       | URL to the build log.                                                                               |
> | `TRAVIS_BUILD_DIR`           | The absolute path to the directory where the repository being built has been copied on the worker.   |
> | `TRAVIS_JOB_WEB_URL`         | URL to the job log.                                                                                  |
> | `TRAVIS_REPO_SLUG`           | The slug (in form: owner_name/repo_name) of the repository currently being built.                    |
> | `TRAVIS_COMMIT`              | The commit that the current build is testing.                                                         |
> | `TRAVIS_BRANCH`             | For push builds, the name of the branch. For PR builds, the name of the branch targeted by the PR.   |
> | `TRAVIS_TAG`                 | If the current build is for a git tag, this variable is set to the tag’s name, otherwise it is empty.|
> | `TRAVIS_PULL_REQUEST_SLUG`   | If the current job is a pull request, the slug of the repository from which the PR originated.        |
> | `TRAVIS_PULL_REQUEST_BRANCH` | If the current job is a pull request, the name of the branch from which the PR originated.            |
> | `TRAVIS_COMMIT_MESSAGE`      | The commit subject and body, unwrapped.                                                               |
> | `TRAVIS_ALLOW_FAILURE`       | Set to `true` if the job is allowed to fail, `false` otherwise.                                       |
> | `TRAVIS_APP_HOST`            | The name of the server compiling the build script.                                                   |
> | `TRAVIS_COMMIT_RANGE`        | The range of commits included in the push or pull request.                                           |
> | `TRAVIS_COMPILER`            | Indicates the compiler used by the current job (such as clang, gcc).                                   |
> | `TRAVIS_DEBUG_MODE`          | Set to `true` if the job is running in debug mode.                                                    |
> | `TRAVIS_DIST`                | Indicates the distribution the current job is running on.                                            |
> | `TRAVIS_EVENT_TYPE`          | Indicates how the build was triggered. Possible values are `push`, `pull_request`, `api`, `cron`.   |
> | `TRAVIS_JOB_ID`              | The ID of the current job used internally by Travis CI.                                              |
> | `TRAVIS_JOB_NAME`            | The job name if it was specified, or `""`.                                                             |
> | `TRAVIS_JOB_NUMBER`          | The number of the current job (such as “4.1”).                                                          |
> | `TRAVIS_OS_NAME`             | On multi-OS builds, this value indicates the platform the job is running on (such as `linux`, `osx`). |
> | `TRAVIS_CPU_ARCH`            | On multi-arch builds, this value indicates the CPU architecture (such as `amd64`, `arm64`).           |
> | `TRAVIS_OSX_IMAGE`           | The `osx_image` value configured in `.travis.yml`.                                                    |
> | `TRAVIS_PULL_REQUEST`        | The pull request number if the current job is a pull request, `false` if it’s not a pull request.    |
> | `TRAVIS_PULL_REQUEST_SHA`    | The commit SHA of the HEAD commit of the PR if the job is a pull request.                            |
> | `TRAVIS_PULL_REQUEST_IS_DRAFT` | Set to `true` if the PR is in a draft state, `false` if it is not.                                    |
> | `TRAVIS_SECURE_ENV_VARS`     | Set to `true` if there are any encrypted environment variables, `false` otherwise.                   |
> | `TRAVIS_SUDO`                | `true` or `false` based on whether `sudo` is enabled.                                                 |
> | `TRAVIS_TEST_RESULT`         | `0` if all commands in the script section have exited with zero, `1` otherwise.                     |
> | `TRAVIS_BUILD_STAGE_NAME`    | The build stage. If a build does not use build stages, this variable is empty.                       |
> | `TRAVIS_JOB_RESTARTED`       | Set to `true` if the build is restarted, `false` otherwise.                                           |
> | `TRAVIS_JOB_RESTARTED_BY`    | The login of the user triggering the restart.                                                         |

Author: ```suggestion
| Variable                     | Description                                                                                           |
|------------------------------|-------------------------------------------------------------------------------------------------------|
| `TRAVIS`                     | Always set to `true` to indicate that the build is running on Travis CI.                              |
| `TRAVIS_BUILD_ID`            | The ID of the current build used internally by Travis CI.                                             |
| `TRAVIS_BUILD_NUMBER`        | The number of the current build. For example: `4`.                                                    |
| `TRAVIS_BUILD_WEB_URL`       | URL to the build log.                                                                                 |
| `TRAVIS_BUILD_DIR`           | The absolute path to the directory where the repository being built has been copied on the worker.    |
| `TRAVIS_JOB_WEB_URL`         | URL to the job log.                                                                                   |
| `TRAVIS_REPO_SLUG`           | The slug (in form: `owner_name/repo_name`) of the repository currently being built.                   |
| `TRAVIS_COMMIT`              | The commit that the current build is testing.                                                         |
| `TRAVIS_BRANCH`              | For push builds, the name of the branch. For PR builds, the name of the branch targeted by the PR.    |
| `TRAVIS_TAG`                 | If the current build is for a git tag, this variable is set to the tag’s name, otherwise it is empty. |
| `TRAVIS_PULL_REQUEST_SLUG`   | If the current job is a pull request, the slug of the repository from which the PR originated.        |
| `TRAVIS_PULL_REQUEST_BRANCH` | If the current job is a pull request, the name of the branch from which the PR originated.            |
| `TRAVIS_COMMIT_MESSAGE`      | The commit subject and body, unwrapped.                                                               |

```

For a comprehensive list of environment variables set by Travis CI for every build, see the [official Travis CI documentation][101].

[101]: https://docs.travis-ci.com/user/environment-variables/#default-environment-variables

{{% /tab %}}
{{% tab "Buddy CI" %}}

> | Variable                                | Description                                                                                           |

Author: ```suggestion
| Environment Variable                                | Description                                                                                           |
```
